app/Http/Controllers/FilmController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\Film\CreateRequest;
+use App\Http\Requests\Film\DeleteRequest;
+use App\Http\Requests\Film\EditRequest;
+use App\Models\Film;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class FilmController extends Controller
+{
+    public function createForm()
+    {
+        return view('films.create');
+    }
+
+    public function editForm(int $id)
+    {
+        $film = Film::query()->findOrFail($id);
+        return view('films.edit', compact('film'));
+    }
+
+    public function deleteForm(int $id)
```

Comment:
наверное просто delete
```suggestion
    public function delete(int $id)
```